Source/WebKit/WebProcess/WebCoreSupport/WebPermissionController.h

```diff
@@ -71,3 +80,7 @@ class WebPermissionController final : public CanMakeWeakPtr<WebPermissionControl
 };
 
 } // namespace WebCore
+
+SPECIALIZE_TYPE_TRAITS_BEGIN(WebKit::WebPermissionController)
```

Comment:
Is this needed, I do not see a downcast from PermissionController?

Reply:
Nope, it was needed in a previous iteration of this patch, but not anymore. Thank you for the catch!